src/views/tutorial-view/components/next-previous/components/directional-link-box/index.tsx

```diff
@@ -1,46 +1,38 @@
 import React from 'react'
-import Link, { LinkProps } from 'next/link'
-import InlineSvg from '@hashicorp/react-inline-svg'
+import classNames from 'classnames'
+import { IconArrowRight16 } from '@hashicorp/flight-icons/svg-react/arrow-right-16'
+import { IconArrowLeft16 } from '@hashicorp/flight-icons/svg-react/arrow-left-16'
+import { IconSliders24 } from '@hashicorp/flight-icons/svg-react/sliders-24'
+import CardLink from 'components/card-link'
+import Text from 'components/text'
+import { DirectionalLinkBoxProps, DirectionOption } from './types'
 import s from './directional-link-box.module.css'
 
-interface DirectionalLinkBoxProps {
-  link: LinkProps
-  label: string
-  title: string
-  direction: DirectionOption
+const IconDict: { [k in DirectionOption]: typeof IconArrowRight16 } = {
```

Comment:
Oh nice, that's a good way of working around the svg icon typing.

Reply:
Yeah i Kinda threw up my arms and put `$TSFixMe` there for a sec... and then was like wait I can throw up my arms in a more "correct" way 🤣